imports/ui/Splash/Splash/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
```

Comment:
It looks like we have both `Splash/Splash/index.js` and `Splash/index.js`. Do you want to pick the one you want to use and have it live at `Splash/index.js`?  Delete the other one.